### ISSUE TITLE
Protect `gh-pages` branch

### ISSUE THREAD

**Issue by KirstieJane** [alan-turing-institute/2017-phd-students#35]
@martintoreilly - what do you think we should do for this repository?

My vote would be to require reviews for the main branch so we can focus on the collaborative working aspect of github, but of course it's much faster to allow everyone to commit to master too (see @nathancunn's super speedy fix of #33!)

**Comment by martintoreilly** [comment]
@KirstieJane Yes, I think we should require reviews for the `main` + `gh_pages` branches. We had some issue with students (write access but not admins) not being able to merge PRs during the session on Friday, so it would be good to sit down for 10 minutes with one of them to work out the right combination of branch protection settings. Do you fancy volunteering for this @nathancunn?
